packages/react/src/components/Modal/Modal.mdx

```diff
@@ -37,3 +37,9 @@ Vi bruker native `autoFocus` på `Textfield` for å fokusere inputen i skjemaet.
 ### `Modal.Header`
 
 <ArgTypes of={Modal.Header} />
+
+## Kjente feil
```

Comment:
This looks to be in the wrong mdx file? Should be in `Link.mdx`?

Reply:
Shoot, context switching doing me dirty. Changing it now